src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1555,43 +1581,45 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </PropertyGroup>
 
     <MSBuild
-        Projects="%(_MSBuildProjectReferenceExistent.Identity)"
-        Targets="GetTargetFrameworkProperties"
+        Projects="@(_MSBuildProjectReferenceCrossTargeting)"
+        Targets="GetReferenceProperties"
```

Comment:
It's "public" API surface so I don't like the initial underscore. I'm not sure I prefer `GetProjectReferenceProperties`. Maybe `GetDesiredProperties`?